tutorials/nginx-ingress-gke/index.md

```diff
@@ -0,0 +1,353 @@
+doesn't understand (i.e., all the requests that are not mapped with an Ingress
+Resource). The default backend exposes two URLs:  
+
+-  `/healthz` that returns 200
+-  `/` that returns 404
+
+In Cloud Shell, deploy the default backend deployment and service.
+
+	kubectl apply -f nginx-default-backend.yaml
+
+You can now see that your default backend services and deployments are configured in the cluster:
+
+	kubectl get deployment nginx-default-backend
+
+	NAME                    DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+	nginx-default-backend   1         1         1            1           29s
+
+	kubectl get service nginx-default-backend
+
+	NAME                    TYPE        CLUSTER-IP     EXTERNAL-IP   PORT(S)   AGE
+	nginx-default-backend   ClusterIP   10.7.250.191   <none>        80/TCP    40s
+
+Looking at the YAML file spec, you can see it is a single replica deployment of
+an nginx default backend container from the Google Cloud Repository.
+
+	cat nginx-default-backend.yaml
+
+You will see the following:
+
+	spec:
+	  replicas: 1
+	  template:
+	    metadata:
+	      labels:
+		app: nginx-default-backend
+	    spec:
+	      terminationGracePeriodSeconds: 60
+	      containers:
+	      - name: default-http-backend
+		image: gcr.io/google_containers/defaultbackend:1.0
+		livenessProbe:
+		  httpGet:
+		    path: /healthz
+		    port: 8080
+		    scheme: HTTP
+		  initialDelaySeconds: 30
+		  timeoutSeconds: 5
+
+# Configure an Ingress Resource
+
+An Ingress Resource object is a collection of L7 rules for routing inbound traffic to Kubernetes services.  Multiple rules can be defined in one Ingress Resource or they can be split up into multiple Ingress Resource manifests. The Ingress Resource also determines which controller to utilize to serve traffic.  This can be set with an annotation, kubernetes.io/ingress.class, in the metadata section of the Ingress Resource.  For the nginx controller, use the value nginx as shown below:
+
+	 annotations: kubernetes.io/ingress.class: nginx
+
+
+On Kubernetes Engine, if no annotation is defined under the metadata, the
+Ingress Resource uses the GCP GCLB L7 load balancer to serve traffic.  This
+method can also be forced by setting  the annotation's value to `gce`.
+
+	annotations: kubernetes.io/ingress.class: gce
+
+You can verify the annotation by viewing the ingress-resource.yaml file and check the annotations under the metadata section as shown below.
+
+	cat ingress-resource.yaml
+
+You will see:
+
+	apiVersion: extensions/v1beta1
+	kind: Ingress
+	metadata:
+	  name: ingress-resource
+	  annotations:
+	    kubernetes.io/ingress.class: nginx
+	spec:
+	  rules:
+	  - http:
+	      paths:
+	      - path: /
+		backend:
+		  serviceName: kuard
+		  servicePort: 80
+
+The kind: Ingress dictates it is an Ingress Resource object.  This Ingress Resource defines an inbound L7 rule for path / to service kuard on port 80.
```

Comment:
"The `kind: Ingress` field indicates that this is an..." (wrap key-value in blackticks)